maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -319,12 +340,13 @@ data class AssertCommand(
 data class AssertConditionCommand(
     val condition: Condition,
     private val timeout: String? = null,
+    val label: String? = null,
 ) : Command {
 
     fun timeoutMs() = timeout?.toLong()
 
     override fun description(): String {
-        return "Assert that ${condition.description()}"
+         return label ?: "Assert that ${condition.description()}"
```

Comment:
```suggestion
        return label ?: "Assert that ${condition.description()}"
```

Style: Indent with 4 spaces